assets/scss/components/stencil/navPages/_navPages.scss

```diff
@@ -67,7 +67,7 @@
 // -----------------------------------------------------------------------------
 
 .navPages {
-    background-color: $navPages-backgroundColor;
+    background-color: stencilColor("navPages-subMenu-backgroundColor");
```

Comment:
Instead of wrapping this variable here, do it here:
https://github.com/bigcommerce/cornerstone/blob/cc83ee3fd449d9386b887a946539d7ac4b62fa5b/assets/scss/settings/stencil/navPages/_settings.scss#L12
where the variable is declared